nifi-extension-bundles/nifi-update-attribute-bundle/nifi-update-attribute-ui/src/main/webapp/js/application.js

```diff
@@ -201,6 +201,9 @@ var ua = {
                 }, {
                     text: 'by action',
                     value: 'action'
+                }, {
+                     text: 'filter everything',
```

Comment:
Following the existing pattern, it seems like it would be better to call this `by any field` or `by any property`.